openapi.yaml

```diff
@@ -4711,16 +4711,32 @@ components:
       example: null
       description: |-
         The namespace the `process_id` is valid for.
+
+        The following options are pre-defined by the openEO API, but additional
+        namespaces may be introduced by back-ends or in a future version of the API.
         
-        By default, the namespace is `null`, which includes all pre-defined processes listed at `GET /processes`.
-        The namespace `user` refers to all user-defined processes listed as `GET /process_graphs`.
-        Additional namespaces may be introduced by back-ends or in a future version of the API.
+        * `null` (default): Checks both user-defined and pre-defined processes,
+           but prefers user-defined processes if both are available.
+           This allows users to add missing pre-defined processes for portability,
+           e.g. common processes from [processes.openeo.org](https://processes.openeo.org)
```

Comment:
Is that URL https://processes.openeo.org stable and future-proof enough to be included in the API spec? Otherwise the `e.g. ...` can be dropped without much loss of information I think.

Reply:
The URL is already used in several places throughout the API. It should be stable enough, except that it doesn't contain a reference to a explicit process version.